lib/vcr/cassette/persisters/file_system.rb

```diff
@@ -55,7 +55,15 @@ def sanitized_file_name_from(file_name)
             file_extension = '.' + parts.pop
           end
 
-          parts.join('.').gsub(/[^[:word:]\-\/]+/, '_') + file_extension.to_s
+          file_name = parts.join('.').gsub(/[^[:word:]\-\/]+/, '_') + file_extension.to_s
+          file_name.downcase! if downcase_cassette_names?
+          file_name
+        end
+
+        def downcase_cassette_names?
+          !!VCR.configuration
+            .default_cassette_options
+            .dig(:persister_options, :downcase_cassette_names)
```

Comment:
Should the RSpec and other integrations be updated to support this option? 

I'm looking at https://github.com/vcr/vcr/blob/master/lib/vcr/test_frameworks/rspec.rb#L47-L48